classes/Cart.php

```diff
@@ -3866,7 +3865,7 @@ public function getSummaryDetails($id_lang = null, $refresh = false)
                         $gift_product['price_wt'] = 0;
                         $gift_product['total_wt'] = 0;
                         $gift_product['total'] = 0;
-                        $gift_product['gift'] = true;
+                        $gift_product['is_gift'] = true;
```

Comment:
To answer your comment https://github.com/PrestaShop/PrestaShop/pull/17179/files#r405261136
The problem is more on this line because `Product::getSummaryDetails` is used elsewhere, so if the array field `gift` becomes `is_gift` it changes the expected data returned by this function no?
Unless this was a bug to use `gift` and the expected result is actually to use `is_gift` ?

Reply:
From what I can see in `Cart::getProducts` it seems like `is_gift` is expected you're right! But then I think there is the same mistake in CartRule https://github.com/PrestaShop/PrestaShop/blob/8415bca965c74ff96e211f987c98b648fa8bd0d8/classes/CartRule.php#L767

Reply:
Ping @zuk3975

Reply:
changed that too. ['gift'] to ['is_gift'].